pages/git-protocol.md

```diff
@@ -21,7 +23,9 @@ A guide for programming within version control.
 * Perform work in a feature branch.
 * Rebase frequently to incorporate upstream changes.
 * Use a [pull request] for code reviews.
+* [Sign your commits with a GPG key](https://help.github.com/categories/gpg/).
 
+[2FA]: https://help.github.com/articles/about-two-factor-authentication/
```

Comment:
I totally get the line length thing - usually I just kick the link itself down to the next line if it's going to go past and have it in-line, even if it goes beyond 80 characters (79 to be exact, since the newline is a character too don't forget); it prevents me from having to jump around and scan the document for the shortcut.

In any case there's nothing wrong here and if something is a more common practice/shortcut, by all means stick with it!